package.json

```diff
@@ -47,18 +47,17 @@
     "fr-FR": "./languages/fr.json"
   },
   "devDependencies": {
-    "@peertube/peertube-types": "^4.2.2",
-    "@tsconfig/node16": "^1.0.3",
-    "@typescript-eslint/eslint-plugin": "^5.33.0",
-    "@typescript-eslint/parser": "^5.33.0",
-    "esbuild": "^0.14.36",
-    "eslint": "^8.21.0",
-    "eslint-config-standard-with-typescript": "^22.0.0",
+    "@peertube/peertube-types": "^7.0.0",
+    "@tsconfig/node18": "^18.2.4",
```

Comment:
The `tsconfig/nodexx` dependency should match the node version supported by the supported Peertube version.
This template is meant for Peertube >= 4.2.0, so it should be Node16.